src/Compilers/Core/Portable/Serialization/ObjectWriter.cs

```diff
@@ -802,7 +802,7 @@ public void WriteEncoding(Encoding? encoding)
             }
             else if (encoding.TryGetEncodingKind(out var kind))
             {
-                WriteByte((byte)((int)TypeCode.FirstWellKnownEncoding + kind));
+                WriteByte((byte)((int)TypeCode.FirstWellKnownEncoding + kind - TextEncodingKind.EncodingUtf8));
```

Comment:
don't quick get what's going on here.  doc maybe?

Reply:
Translating one enum range to another.